keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -882,81 +819,66 @@ See Drawbacks section
-
-[existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
--->
+No
 
 ###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
 
-The apiservers will increase the memory usage because they have to keep a local informer with the new objects ServiceCIDR and IPAddress.
-
-### Troubleshooting
+The apiservers and kube-controller manager will increase the memory usage because they have to keep a local informer with the new objects ServiceCIDR and IPAddress.
 
-<!--
-This section must be completed when targeting beta to a release.
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
 
-For GA, this section is required: approvers should be able to confirm the
-previous answers based on experience in the field.
+No
 
-The Troubleshooting section currently serves the `Playbook` role. We may consider
-splitting it into a dedicated `Playbook` document (potentially with some monitoring
-details). For now, we leave it here.
--->
+### Troubleshooting
 
 ###### How does this feature react if the API server and/or etcd is unavailable?
 
-The feature depends on the API server to work.
+The feature depends on the API server to work, however, the components consuming the existing objects will be unaffected.
 
 ###### What are other known failure modes?
 
-<!--
-For each of them, fill in the following information by copying the below template:
-  - [Failure mode brief description]
-    - Detection: How can it be detected via metrics? Stated another way:
-      how can an operator troubleshoot without logging into a master or worker node?
-    - Mitigations: What can be done to stop the bleeding, especially for already
-      running user workloads?
-    - Diagnostics: What are the useful log messages and their required logging
-      levels that could help debug the issue?
-      Not required until feature graduated to beta.
-    - Testing: Are there any tests for failure mode? If not, describe why.
--->
+- Can not create Service with ClusterIP
+  - Detection: via metrics that record errors and latency to allocate an Cluster IP, the allocator also log slow operations in `pkg/registry/core/service/ipallocator/ipallocator.go`
+> 	trace := utiltrace.New("allocate dynamic ClusterIP address")
+>	defer trace.LogIfLong(500 * time.Millisecond)
+  - Mitigations: already running workloads will not be affected, if no new services can be created first check if is because there
+are not available IPs checking the metrics with the number of available IPs
+  - Diagnostics: The ServiceCIDR object has conditions in state that can help to understand if there is some problems with
+the available ranges. The kube-apiserve metrics shows the repair IP and ip allocation errors. 
+  - Testing: See integration tests added
```

Comment:
Either in this section or in the upgrade section, can you describe a potential problem during upgrade, when you have two apiservers (one with this new feature enabled and one without) working in parallel. The new one will work against the new API objects as the source of truth, but the old one will still work of the in-memory one, which can potentially differ as you pointed out earlier in this document. I believe the answer is that the repair loop will eventually settle this situation, but I'd appreciate calling it out here explicitly.

Reply:
expanded the `Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?` section, PTAL

Reply:
I think there's still a potential issue here.  https://github.com/kubernetes/enhancements/pull/4645#discussion_r1637228581

Reply:
solved the issue and added a test validating it https://github.com/kubernetes/kubernetes/pull/122047/commits/b29cbfaf1664acf84cb08cf1f9d8bc6cd757ebb1